extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -733,6 +734,9 @@ export class AmpA4A extends AMP.BaseElement {
           // Preload any fonts.
           (creativeMetaDataDef.customStylesheets || []).forEach(font =>
               this.preconnect.preload(font.href));
+          // Preload any AMP images.
+          (creativeMetaDataDef.images || []).forEach(image =>
+              isSecureUrl(image) && this.preconnect.preload(image));
```

Comment:
Who controls the origin of these images? How do we know we're not preconneting to a tracking service?